docs/README.zh.md

```diff
@@ -0,0 +1,52 @@
+# 学习资源库 (Learning Resources Repository)
+
+[English](../README.md) | [中文](README.zh.md)
+
+欢迎来到 **Learning Resources Repository**！此仓库旨在为您提供各个领域的精选资源，帮助您的个人和职业发展。无论您是学生、专业人士，还是仅仅对某个领域感兴趣，这里都有丰富的资料可以帮助您提升技能和知识。
+
+## 📚 目录 (Contents)
+
+1. [前端开发 (Frontend Development)](#-frontend-development)
+2. [后端开发 (Backend Development)](#-backend-development)
+3. [个人发展 (Personal Development)](#-personal-development)
```

Comment:
There should be a slight change in the links. None of the links in the Chinese version are working correctly; most lead to 404 error page.

Try figuring it out yourself and please feel free to reach out if you want my help. 
ps: I don't want to spoon-feed the corrections to you.😉